### ISSUE TITLE
Transfer open issues from previous github issue tracker

### ISSUE THREAD

**Issue by progfolio** [emacs-exwm/exwm#50]
I'm willing to do some triage/manual transferring if we think it's worth it.

**Comment by minad** [comment]
Yes, this would be appreciated. However note that only @medranocalvo can close issues on the original issue tracker. But when we add a note/link to each manually transferred issue, @medranocalvo can search for these issues and close them when he finds time.

**Comment by progfolio** [comment]
> @progfolio, thank you again, that will be helpful. I defined some labels in @ch11ng:exwm, are you able to label issues over there? If so I can add more labels if that helps.

I do not have write permissions on either repository, so I won't be able to apply labels to any issues.

**Comment by Stebalien** [comment]
@progfolio I've given you triage access (you'll need to accept the invite).
